shared/guide/system/software/updating/security_patches_up_to_date.rule

```diff
@@ -36,6 +36,8 @@ references:
     srg: SRG-OS-000480-GPOS-00227
     stigid@rhel7: "020260"
 
+oval_external_content: https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2
```

Comment:
This seems Red Hat specific. What happens if/when someone adds content for other communities, e.g. the Ubuntu OVAL feed @ https://people.canonical.com/~ubuntu-security/oval/

Reply:
@shawndwells true. This is being worked as a separate effort with jinja templating.

Reply:
This is a perfect case for a jinja example. I will get that working in #2783 once this is merged.